SPECS/cloud-hypervisor-cvm/cloud-hypervisor-cvm.spec

```diff
@@ -0,0 +1,229 @@
+# This spec file assumes you're building on an environment where:
+# * You have access to the internet during the build
+# * You have rustup installed on your system
+# * You have both x86_64-unknown-linux-gnu and x86_64-unknown-linux-musl, or
+# * You have both aarch64-unknown-linux-gnu and aarch64-unknown-linux-musl
+#   targets installed.
+
+%define using_rustup 0
+%define using_musl_libc 0
+%define using_vendored_crates 1
+
+# Generated by update script
+%global gitver   38.0
+%global gitnum   72
+%global gitrev   8f7209bdc
+%global clh_name cloud-hypervisor
+%global clh_type cvm
+
+Name:           %{clh_name}-%{clh_type}
+Summary:        Cloud Hypervisor CVM is an open source Virtual Machine Monitor (VMM) that enables running SEV SNP enabled VMs on top of MSHV using the IGVM file format as payload.
+Version:        %{gitver}.%{gitnum}
+Release:        1000.%{gitrev}%{?dist}
+License:        ASL 2.0 or BSD-3-clause
+Group:          Applications/System
+URL:            https://github.com/cloud-hypervisor/cloud-hypervisor
+Source0:        https://github.com/microsoft/cloud-hypervisor/archive/refs/tags/msft/v%{version}.tar.gz#/%{clh_name}-%{version}.tar.gz
+%if 0%{?using_vendored_crates}
+Source1:        %{clh_name}-%{version}-vendor.tar.gz
+Source2:        config.toml
+%endif
+
+Conflicts: cloud-hypervisor
+
+BuildRequires:  binutils
+BuildRequires:  gcc
+BuildRequires:  git
+BuildRequires:  glibc-devel
+BuildRequires:  openssl-devel
+
+%if ! 0%{?using_rustup}
+BuildRequires:  rust >= 1.62.0
+BuildRequires:  cargo >= 1.62.0
+%endif
+
+Requires: bash
+Requires: glibc
+Requires: libgcc
+Requires: libcap
+
+ExclusiveArch:  x86_64
+
+%ifarch x86_64
+%define rust_def_target x86_64-unknown-linux-gnu
+%define cargo_pkg_feature_opts --no-default-features --features "mshv,kvm,sev_snp,igvm"
+%endif
+%ifarch aarch64
+%define rust_def_target aarch64-unknown-linux-gnu
+%define cargo_pkg_feature_opts --all
+%endif
+
+%if 0%{?using_musl_libc}
+%ifarch x86_64
+%define rust_musl_target x86_64-unknown-linux-musl
+%endif
+%ifarch aarch64
+%define rust_musl_target aarch64-unknown-linux-musl
+%endif
+%endif
+
+%if 0%{?using_vendored_crates}
+%define cargo_offline --offline
+%endif
+
+%description
+Cloud Hypervisor is an open source Virtual Machine Monitor (VMM) that runs on top of KVM. The project focuses on exclusively running modern, cloud workloads, on top of a limited set of hardware architectures and platforms. Cloud workloads refers to those that are usually run by customers inside a cloud provider. For our purposes this means modern Linux* distributions with most I/O handled by paravirtualised devices (i.e. virtio), no requirement for legacy devices and recent CPUs and KVM.
+
+%prep
+
+%autosetup -p1 -n %{clh_name}-%{version}
```

Comment:
in the regular CLH package we use %setup. Best to align. In general we can align this block a bit more with its regular CLH counterpart.